hooks/pre-commit.sh

```diff
@@ -1,10 +1,6 @@
 #!/bin/bash
 set -eo pipefail
 
-if [ "$GITHUB_ACTIONS" != true ]; then
-  vale docusaurus
-fi
```

Comment:
Why are we doing this? 🤔

Reply:
this breaks the workflow (can't commit) if you don't have a specific vale version, which is no longer supported. Leaving this for @testableapple to solve properly.